sdk/consumption/azure-resourcemanager-consumption/src/test/java/com/azure/resourcemanager/consumption/ConsumptionTests.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.consumption;
+
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.core.util.Configuration;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.consumption.models.Budget;
+import com.azure.resourcemanager.consumption.models.ReservationRecommendation;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ConsumptionTests extends TestBase {
+
+    @Test
+    @DoNotRecord(skipInPlayback = true)
+    public void availabilityTest() {
```

Comment:
If we skip the test in playback mode, do we need to run the test locally when generate SDK for consumption?

Reply:
Yeah. There is configure for LIVE run in CI. But current no one get bandwidth to investigate and adapt.

Reply:
Add issue https://github.com/Azure/azure-sdk-for-java/issues/25915

Feel free to pick up.